InvenTree/InvenTree/management/commands/prerender.py

```diff
@@ -0,0 +1,61 @@
+"""
+Custom management command to prerender files
+"""
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.template.loader import render_to_string
+from django.utils.module_loading import import_string
+from django.http.request import HttpRequest
+from django.utils.translation import override as lang_over
+
+import os
+
+
+def render_file(file_name, source, target, locales, ctx):
+    """ renders a file into all provided locales """
+    for locale in locales:
+        target_file = os.path.join(target, locale + '.' + file_name)
+        with open(target_file, 'w') as localised_file:
+            with lang_over(locale):
+                renderd = render_to_string(os.path.join(source, file_name), ctx)
+                localised_file.write(renderd)
+
+
+class Command(BaseCommand):
+    """
+    django command to prerender files
+    """
+
+    def handle(self, *args, **kwargs):
+        # static directorys
+        LC_DIR = settings.LOCALE_PATHS[0]
+        SOURCE_DIR = settings.STATICFILES_I18_SRC
+        TARTGET_DIR = settings.STATICFILES_I18_TRG
+
+        # ensure static directory exists
+        if not os.path.exists(TARTGET_DIR):
+            os.mkdir(TARTGET_DIR)
```

Comment:
Should be `os.makedirs(TARGET_DIR, exist_ok=True)`

os.mkdir fails on my system as I don't already have "static_i18n" directory